Started UtBot Contest, classfileDir = /var/benchmarks/projects/guava/guava/target/classes, classpathString=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, outputDir=/home/utbot_mocks/temp/testcases, mocks=Other classes: Mockito
01:48:10.031 | INFO  | Started: warmup: kotlin reflection :: init
01:48:10.121 | INFO  | Finished (in 0.089 sec): warmup: kotlin reflection :: init 
[INIT]
>> [RUN] com.google.common.base.Throwables 120
01:48:10.129 | DEBUG | -----------------------------------------------------------------------------
01:48:10.130 | INFO  | Contest.runGeneration: Time budget: 120000 ms, Generation timeout=102000 ms, classpath=/var/benchmarks/projects/guava/guava/target/classes:/var/benchmarks/projects/guava/guava/target/guava-26.0-jre.jar:/var/benchmarks/projects/./guava/guava/target/dependency/checker-qual-2.5.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/error_prone_annotations-2.1.3.jar:/var/benchmarks/projects/guava/guava/target/dependency/j2objc-annotations-1.1.jar:/var/benchmarks/projects/guava/guava/target/dependency/jsr305-3.0.2.jar:/var/benchmarks/projects/guava/guava/target/dependency/animal-sniffer-annotations-1.14.jar, methodNameFilter=null
01:48:10.131 | INFO  | ClassUnderTest[ FQN: com.google.common.base.Throwables
    classfileDir: /var/benchmarks/projects/guava/guava/target/classes
    testClassSimpleName: ThrowablesTest
    generatedTestFile: /home/utbot_mocks/temp/testcases/com/google/common/base/ThrowablesTest.java
    generatedTestsSourcesDir: /home/utbot_mocks/temp/testcases
]
01:48:10.155 | INFO  | Started: class com.google.common.base.Throwables
01:48:10.156 | INFO  | Started: preparation class class com.google.common.base.Throwables: kotlin reflection :: run
01:48:10.198 | INFO  | Finished (in 0.041 sec): preparation class class com.google.common.base.Throwables: kotlin reflection :: run 
01:48:10.199 | INFO  | Started: 2nd optional soot initialization
01:48:10.200 | INFO  | Finished (in 0.001 sec): 2nd optional soot initialization 
01:48:10.208 | DEBUG |  ... 
01:48:10.221 | INFO  | Started: method com.google.common.base.Throwables.getRootCause(Throwable)
01:48:10.222 | INFO  |  -- Remaining time budget: 101920 ms, #remaining_methods: 20, budget for method: 5096 ms, solver timeout: 1000 ms, budget for symbolic execution: 3896 ms, budget for concrete execution: 1200 ms,  -- 
01:48:10.550 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)1
01:48:10.914 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)2
01:48:11.208 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)3
01:48:11.886 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)4
01:48:12.390 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)5
01:48:12.820 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getRootCause(Throwable)6
01:48:14.113 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getRootCause(Throwable)
01:48:15.314 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getRootCause(Throwable)
01:48:15.859 | INFO  | Finished (in 5.637 sec): method com.google.common.base.Throwables.getRootCause(Throwable) 
<StatsForMethod> :
	#generatedTC=6
	WITH NO EXCEPTIONS
01:48:15.953 | DEBUG |  ... 
01:48:15.954 | INFO  | Started: method com.google.common.base.Throwables.propagate(Throwable)
01:48:15.955 | INFO  |  -- Remaining time budget: 96176 ms, #remaining_methods: 19, budget for method: 5061 ms, solver timeout: 1000 ms, budget for symbolic execution: 3861 ms, budget for concrete execution: 1200 ms,  -- 
01:48:15.967 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)1
01:48:16.644 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)2
01:48:16.923 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)3
01:48:17.099 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagate(Throwable)4
01:48:17.099 | INFO  | Finished (in 1.144 sec): method com.google.common.base.Throwables.propagate(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
01:48:17.124 | DEBUG |  ... 
01:48:17.124 | INFO  | Started: method com.google.common.base.Throwables.throwIfUnchecked(Throwable)
01:48:17.124 | INFO  |  -- Remaining time budget: 95006 ms, #remaining_methods: 18, budget for method: 5278 ms, solver timeout: 1000 ms, budget for symbolic execution: 4078 ms, budget for concrete execution: 1200 ms,  -- 
01:48:17.132 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)1
01:48:17.391 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)2
01:48:17.596 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)3
01:48:17.754 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfUnchecked(Throwable)4
01:48:17.755 | INFO  | Finished (in 0.630 sec): method com.google.common.base.Throwables.throwIfUnchecked(Throwable) 
<StatsForMethod> :
	#generatedTC=4
	WITH NO EXCEPTIONS
01:48:17.757 | DEBUG |  ... 
01:48:17.757 | INFO  | Started: method com.google.common.base.Throwables.getCausalChain(Throwable)
01:48:17.757 | INFO  |  -- Remaining time budget: 94373 ms, #remaining_methods: 17, budget for method: 5551 ms, solver timeout: 1000 ms, budget for symbolic execution: 4351 ms, budget for concrete execution: 1200 ms,  -- 
01:48:17.769 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)1
01:48:18.981 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)2
01:48:22.109 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.getCausalChain(Throwable)
01:48:22.840 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCausalChain(Throwable)3
01:48:23.310 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.getCausalChain(Throwable)
01:48:23.482 | INFO  | Finished (in 5.724 sec): method com.google.common.base.Throwables.getCausalChain(Throwable) 
<StatsForMethod> :
	#generatedTC=3
	WITH NO EXCEPTIONS
01:48:23.630 | DEBUG |  ... 
01:48:23.631 | INFO  | Started: method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)
01:48:23.631 | INFO  |  -- Remaining time budget: 88500 ms, #remaining_methods: 16, budget for method: 5531 ms, solver timeout: 1000 ms, budget for symbolic execution: 4331 ms, budget for concrete execution: 1200 ms,  -- 
01:48:23.643 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
01:48:24.007 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)1
01:48:24.144 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)2
01:48:24.643 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class)3
01:48:24.689 | INFO  | Finished (in 1.058 sec): method com.google.common.base.Throwables.throwIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=3
	FAILED 1 time(s) with 1 different exception(s)"
01:48:24.700 | DEBUG |  ... 
01:48:24.701 | INFO  | Started: method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)
01:48:24.701 | INFO  |  -- Remaining time budget: 87429 ms, #remaining_methods: 15, budget for method: 5828 ms, solver timeout: 1000 ms, budget for symbolic execution: 4628 ms, budget for concrete execution: 1200 ms,  -- 
01:48:24.706 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
01:48:24.739 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)1
01:48:25.500 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)2
01:48:25.835 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)3
01:48:26.017 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class)4
01:48:26.018 | INFO  | Finished (in 1.317 sec): method com.google.common.base.Throwables.propagateIfInstanceOf(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=4
	FAILED 1 time(s) with 1 different exception(s)"
01:48:26.050 | DEBUG |  ... 
01:48:26.051 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class)
01:48:26.051 | INFO  |  -- Remaining time budget: 86079 ms, #remaining_methods: 14, budget for method: 6148 ms, solver timeout: 1000 ms, budget for symbolic execution: 4948 ms, budget for concrete execution: 1200 ms,  -- 
01:48:26.056 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
01:48:27.433 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)1
01:48:27.460 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)2
01:48:27.862 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)3
01:48:28.013 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)4
01:48:28.201 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)5
01:48:28.405 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class)6
01:48:28.405 | INFO  | Finished (in 2.354 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=6
	FAILED 1 time(s) with 1 different exception(s)"
01:48:28.466 | DEBUG |  ... 
01:48:28.466 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable)
01:48:28.467 | INFO  |  -- Remaining time budget: 83664 ms, #remaining_methods: 13, budget for method: 6435 ms, solver timeout: 1000 ms, budget for symbolic execution: 5235 ms, budget for concrete execution: 1200 ms,  -- 
01:48:28.474 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)1
01:48:28.589 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)2
01:48:29.147 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)3
01:48:29.283 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)4
01:48:29.468 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable)5
01:48:29.468 | INFO  | Finished (in 1.001 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable) 
<StatsForMethod> :
	#generatedTC=5
	WITH NO EXCEPTIONS
01:48:29.492 | DEBUG |  ... 
01:48:29.493 | INFO  | Started: method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)
01:48:29.493 | INFO  |  -- Remaining time budget: 82637 ms, #remaining_methods: 12, budget for method: 6886 ms, solver timeout: 1000 ms, budget for symbolic execution: 5686 ms, budget for concrete execution: 1200 ms,  -- 
01:48:29.500 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
01:48:30.831 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)1
01:48:31.356 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)2
01:48:32.771 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)3
01:48:33.067 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class)4
01:48:33.067 | INFO  | Finished (in 3.574 sec): method com.google.common.base.Throwables.propagateIfPossible(Throwable, Class, Class) 
<StatsForMethod> :
	#generatedTC=4
	FAILED 1 time(s) with 1 different exception(s)"
01:48:33.188 | DEBUG |  ... 
01:48:33.189 | INFO  | Started: method com.google.common.base.Throwables.getCauseAs(Throwable, Class)
01:48:33.189 | INFO  |  -- Remaining time budget: 78942 ms, #remaining_methods: 11, budget for method: 7176 ms, solver timeout: 1000 ms, budget for symbolic execution: 5976 ms, budget for concrete execution: 1200 ms,  -- 
01:48:33.194 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ChildProcessError: Error in the child process |> java.lang.IllegalAccessException: java.lang.Class
	at sun.misc.Unsafe.allocateInstance(Native Method)
	at com.huawei.utbot.framework.util.EngineUtilsKt.getAnyInstance(EngineUtils.kt:61)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructObject(MockValueConstructor.kt:158)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.construct(MockValueConstructor.kt:124)
	at com.huawei.utbot.framework.concrete.MockValueConstructor.constructMethodParameters(MockValueConstructor.kt:101)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:126)
	at com.huawei.utbot.framework.concrete.UtExecutionInstrumentation.invoke(UtExecutionInstrumentation.kt:96)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.loop(ChildProcess.kt:114)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt:76)
	at com.huawei.utbot.instrumentation.process.ChildProcessKt.main(ChildProcess.kt)

	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:391) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
01:48:33.298 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)1
01:48:33.704 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)2
01:48:33.797 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)3
01:48:34.354 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)4
01:48:34.809 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)5
01:48:35.377 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)6
01:48:35.830 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getCauseAs(Throwable, Class)7
01:48:36.081 | INFO  | Finished (in 2.892 sec): method com.google.common.base.Throwables.getCauseAs(Throwable, Class) 
<StatsForMethod> :
	#generatedTC=7
	FAILED 1 time(s) with 1 different exception(s)"
01:48:36.194 | DEBUG |  ... 
01:48:36.195 | INFO  | Started: method com.google.common.base.Throwables.getStackTraceAsString(Throwable)
01:48:36.195 | INFO  |  -- Remaining time budget: 75935 ms, #remaining_methods: 10, budget for method: 7593 ms, solver timeout: 1000 ms, budget for symbolic execution: 6393 ms, budget for concrete execution: 1200 ms,  -- 
01:48:36.207 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)1
01:48:36.970 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getStackTraceAsString(Throwable)2
01:48:36.971 | INFO  | Finished (in 0.775 sec): method com.google.common.base.Throwables.getStackTraceAsString(Throwable) 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
01:48:37.012 | DEBUG |  ... 
01:48:37.013 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTrace(Throwable)
01:48:37.013 | INFO  |  -- Remaining time budget: 75117 ms, #remaining_methods: 9, budget for method: 8346 ms, solver timeout: 1000 ms, budget for symbolic execution: 7146 ms, budget for concrete execution: 1200 ms,  -- 
01:48:44.160 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTrace(Throwable)
01:48:45.361 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.lazyStackTrace(Throwable)
01:48:45.425 | INFO  | Finished (in 8.412 sec): method com.google.common.base.Throwables.lazyStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
01:48:45.885 | DEBUG |  ... 
01:48:45.886 | INFO  | Started: method com.google.common.base.Throwables.lazyStackTraceIsLazy()
01:48:45.886 | INFO  |  -- Remaining time budget: 66245 ms, #remaining_methods: 8, budget for method: 8280 ms, solver timeout: 1000 ms, budget for symbolic execution: 7080 ms, budget for concrete execution: 1200 ms,  -- 
01:48:45.890 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.lazyStackTraceIsLazy()1
01:48:52.966 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.lazyStackTraceIsLazy()
01:48:53.116 | INFO  | Finished (in 7.230 sec): method com.google.common.base.Throwables.lazyStackTraceIsLazy() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
01:48:53.521 | DEBUG |  ... 
01:48:53.521 | INFO  | Started: method com.google.common.base.Throwables.jlaStackTrace(Throwable)
01:48:53.522 | INFO  |  -- Remaining time budget: 58609 ms, #remaining_methods: 7, budget for method: 8372 ms, solver timeout: 1000 ms, budget for symbolic execution: 7172 ms, budget for concrete execution: 1200 ms,  -- 
01:48:53.755 | INFO  | Finished (in 0.233 sec): method com.google.common.base.Throwables.jlaStackTrace(Throwable) 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
01:48:53.766 | DEBUG |  ... 
01:48:53.767 | INFO  | Started: method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
01:48:53.767 | INFO  |  -- Remaining time budget: 58364 ms, #remaining_methods: 6, budget for method: 9727 ms, solver timeout: 1000 ms, budget for symbolic execution: 8527 ms, budget for concrete execution: 1200 ms,  -- 
01:48:53.781 | ERROR | Symbolic execution FAILED
com.huawei.utbot.instrumentation.util.ReadingFromKryoException: Failure reading from kryo
	at com.huawei.utbot.instrumentation.ConcreteExecutor.executeAsync(ConcreteExecutor.kt:392) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$executeAsync$1.invokeSuspend(ConcreteExecutor.kt) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) [utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) [utbot-junit-contest-1.0.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_312]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180) [?:1.8.0_312]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_312]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_312]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
Caused by: com.esotericsoftware.kryo.KryoException: Encountered unregistered class ID: 30
	at com.esotericsoftware.kryo.util.DefaultClassResolver.readClass(DefaultClassResolver.java:159) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClass(Kryo.java:758) ~[utbot-junit-contest-1.0.jar:?]
	at com.esotericsoftware.kryo.Kryo.readClassAndObject(Kryo.java:869) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.util.KryoHelper.readCommand(KryoHelper.kt:46) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1$1.invokeSuspend(ConcreteExecutor.kt:256) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:274) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:84) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) ~[utbot-junit-contest-1.0.jar:?]
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:253) ~[utbot-junit-contest-1.0.jar:?]
	at com.huawei.utbot.instrumentation.ConcreteExecutor$restartIfNeeded$$inlined$bracket$lambda$1$1.invoke(ConcreteExecutor.kt:101) ~[utbot-junit-contest-1.0.jar:?]
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30) ~[utbot-junit-contest-1.0.jar:?]
01:48:54.116 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])1
01:49:02.294 | INFO  | |> Starting concrete execution for remaining state: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
01:49:03.495 | INFO  | (X) Cancelling concrete execution: com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[])
01:49:03.867 | INFO  | Finished (in 10.100 sec): method com.google.common.base.Throwables.invokeAccessibleNonThrowingMethod(Method, Object, Object[]) 
<StatsForMethod> :
	#generatedTC=1
	FAILED 1 time(s) with 1 different exception(s)"
01:49:03.962 | DEBUG |  ... 
01:49:03.962 | INFO  | Started: method com.google.common.base.Throwables.getJLA()
01:49:03.962 | INFO  |  -- Remaining time budget: 48168 ms, #remaining_methods: 5, budget for method: 9633 ms, solver timeout: 1000 ms, budget for symbolic execution: 8433 ms, budget for concrete execution: 1200 ms,  -- 
01:49:12.300 | INFO  | Finished (in 8.338 sec): method com.google.common.base.Throwables.getJLA() 
<StatsForMethod> : SUSPICIOUS
	#generatedTC=0
	WITH NO EXCEPTIONS
01:49:12.386 | DEBUG |  ... 
01:49:12.386 | INFO  | Started: method com.google.common.base.Throwables.getGetMethod()
01:49:12.386 | INFO  |  -- Remaining time budget: 39744 ms, #remaining_methods: 4, budget for method: 9936 ms, solver timeout: 1000 ms, budget for symbolic execution: 8736 ms, budget for concrete execution: 1200 ms,  -- 
01:49:12.415 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getGetMethod()1
01:49:19.763 | INFO  | Finished (in 7.377 sec): method com.google.common.base.Throwables.getGetMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
01:49:20.176 | DEBUG |  ... 
01:49:20.177 | INFO  | Started: method com.google.common.base.Throwables.getSizeMethod()
01:49:20.177 | INFO  |  -- Remaining time budget: 31953 ms, #remaining_methods: 3, budget for method: 10651 ms, solver timeout: 1000 ms, budget for symbolic execution: 9451 ms, budget for concrete execution: 1200 ms,  -- 
01:49:20.186 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getSizeMethod()1
01:49:27.400 | INFO  | Finished (in 7.223 sec): method com.google.common.base.Throwables.getSizeMethod() 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
01:49:27.810 | DEBUG |  ... 
01:49:27.811 | INFO  | Started: method com.google.common.base.Throwables.getJlaMethod(String, Class)
01:49:27.811 | INFO  |  -- Remaining time budget: 24319 ms, #remaining_methods: 2, budget for method: 12159 ms, solver timeout: 1000 ms, budget for symbolic execution: 10959 ms, budget for concrete execution: 1200 ms,  -- 
01:49:27.839 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.getJlaMethod(String, Class)1
01:49:35.026 | INFO  | Finished (in 7.215 sec): method com.google.common.base.Throwables.getJlaMethod(String, Class) 
<StatsForMethod> :
	#generatedTC=1
	WITH NO EXCEPTIONS
01:49:35.282 | DEBUG |  ... 
01:49:35.283 | INFO  | Started: method com.google.common.base.Throwables.<init>()
01:49:35.283 | INFO  |  -- Remaining time budget: 16847 ms, #remaining_methods: 1, budget for method: 16847 ms, solver timeout: 1000 ms, budget for symbolic execution: 15647 ms, budget for concrete execution: 1200 ms,  -- 
01:49:35.289 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()1
01:49:35.311 | DEBUG | --new testCase collected, to generate: testCom.google.common.base.Throwables.<init>()2
01:49:35.312 | INFO  | Finished (in 0.028 sec): method com.google.common.base.Throwables.<init>() 
<StatsForMethod> :
	#generatedTC=2
	WITH NO EXCEPTIONS
01:49:35.319 | INFO  | Started: Flushing tests for [Throwables] on disk
01:49:35.677 | INFO  | File size for ThrowablesTest: 49 KB
01:49:35.679 | INFO  | Finished (in 0.359 sec): Flushing tests for [Throwables] on disk 
01:49:35.680 | INFO  | Finished (in 85.523 sec): class com.google.common.base.Throwables 
<StatsForClass> :
	canceled by timeout = false
	#methods = 20, 
	#methods started symbolic exploration = 20
	#methods with at least one TC = 17
	#methods with exceptions = 6
	#generated TC = 55
	#coverage = null
[READY]
